application/app/models/project.rb

```diff
@@ -5,10 +5,8 @@ class Project < ApplicationDiskRecord
   include FileStatusSummary
   include LoggingCommon
 
-  MAX_REPO_URLS = 20
-
   REQUIRED_ATTRIBUTES = %w[id name download_dir creation_date].freeze
-  ATTRIBUTES = (REQUIRED_ATTRIBUTES + %w[repo_urls]).freeze
+  ATTRIBUTES = REQUIRED_ATTRIBUTES
```

Comment:
This change was part of these changes, but in the end, I am using the files to get the same data